api/api_public_blockchain.go

```diff
@@ -150,14 +150,58 @@ func (s *PublicBlockChainAPI) GetAccount(ctx context.Context, address common.Add
 	return serAcc, state.Error()
 }
 
+// rpcMarshalHeader converts the given header to the RPC output.
+func (s *PublicBlockChainAPI) rpcMarshalHeader(header *types.Header) map[string]interface{} {
+	fields := RPCMarshalHeader(header, s.b.ChainConfig().IsEthTxTypeForkEnabled(header.Number))
+	return fields
+}
+
+// RPCMarshalHeader converts the given header to the RPC output that includes the baseFeePerGas field.
+func RPCMarshalHeader(head *types.Header, isEnabledEthTxTypeFork bool) map[string]interface{} {
+	result := map[string]interface{}{
+		"parentHash":       head.ParentHash,
+		"reward":           head.Rewardbase,
+		"stateRoot":        head.Root,
+		"transactionsRoot": head.TxHash,
+		"receiptsRoot":     head.ReceiptHash,
+		"logsBloom":        head.Bloom,
+		"blockScore":       (*hexutil.Big)(head.BlockScore),
+		"number":           (*hexutil.Big)(head.Number),
+		"gasUsed":          hexutil.Uint64(head.GasUsed),
+		"timestamp":        (*hexutil.Big)(head.Time),
+		"timestampFoS":     hexutil.Uint64(head.TimeFoS),
+		"extraData":        hexutil.Bytes(head.Extra),
+		"governanceData":   hexutil.Bytes(head.Governance),
+		"hash":             head.Hash(),
+	}
+
+	if len(head.Vote) != 0 {
+		result["voteData"] = hexutil.Bytes(head.Vote)
+	}
+
+	if isEnabledEthTxTypeFork {
+		result["baseFeePerGas"] = hexutil.Uint64(params.BaseFee)
```

Comment:
```suggestion
		result["baseFeePerGas"] = (*hexutil.Big)(params.BaseFee)
```

Reply:
I modified type to *hexutil.Big in https://github.com/klaytn/klaytn/pull/1183/commits/475ba87330f9b9a11840608abb7b5f95972777e5